src/libraries/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -37,20 +39,20 @@ public sealed partial class Collection : System.Collections.ICollection, System.
         bool System.Collections.IList.IsFixedSize { get { throw null; } }
         bool System.Collections.IList.IsReadOnly { get { throw null; } }
         object System.Collections.IList.this[int index] { get { throw null; } set { } }
-        public void Add(object Item, string Key = null, object Before = null, object After = null) { }
+        public void Add(object? Item, string? Key = null, object? Before = null, object? After = null) { }
         public void Clear() { }
         public bool Contains(string Key) { throw null; }
         public System.Collections.IEnumerator GetEnumerator() { throw null; }
         public void Remove(int Index) { }
         public void Remove(string Key) { }
```

Comment:
Same question, since we can add a `null` key, can we remove it?

Reply:
It looks like yes, this should be `string?`.